src/database/database.service.ts

```
@@ -0,0 +1,83 @@
+                else DatabaseService.logger.log('DB Connection closed');
+            });
+        }
+    }
+
+    private connect(dbPath: string): Promise<void> {
+        return new Promise((resolve, reject) => {
+            this.db = new sqlite3.Database(dbPath, (err) => {
+                if (err) {
+                    DatabaseService.logger.error(`❌ Connection failed: ${err.message}`);
+                    reject(err);
+                } else {
```

ESLint

[error] Replace `················` with `········`

(prettier/prettier)